User-Help/GlobalTrustLevels.md

```diff
@@ -0,0 +1,138 @@
+| **Suspend specific user permissions**     | n/a                                                                                  | n/a                                                                                                  | n/a                                              | n/a                                                                                                 | n/a                                                                               | Full access                                                    |
+| **Sent private messages to users**        | n/a                                                                                  | n/a                                                                                                  | n/a                                              | n/a                                                                                                 | n/a                                                                               | Full access                                                    |
+| **Annotate users**                        | n/a                                                                                  | n/a                                                                                                  | n/a                                              | n/a                                                                                                 | n/a                                                                               | Full access                                                    |
+| **View private messages and annotations** | n/a                                                                                  | n/a                                                                                                  | n/a                                              | n/a                                                                                                 | n/a                                                                               | Full access                                                    |
+| **Access to all other moderation tools**  | n/a                                                                                  | n/a                                                                                                  | n/a                                              | n/a                                                                                                 | n/a                                                                               | Full access                                                    |
+
+
+
+# Glossary of terms
+
+### Prerequisites
+
+Minimum activity required to reach a given trust level.
+
+### Posts
+
+Typically posts are questions. Depending on the category, however, they can also include other formats like blogs or wikis.
+
+### Responses 
```

Comment:
Did we use the term "response" in here?

Reply:
Re this and the prior comment: I was using "post" as a generic term to refer to the original entry (question, contest, etc.) and "response" as the parallel for any non-comment replies (answers, submissions, etc.)

If anything except a comment is considered a post, do we have explicit vocabulary to refer to an initial post vs. response posts?

Reply:
I think we've been calling the main posts, i.e. the things that show up in the list of posts (questions, blog posts, etc), "top-level posts".  That might not be a great externally-facing name, though.

A post is any of the longer-form items -- question, answer, article (blog post, wiki page, etc), maybe others.  If you go to the Q&A category you see a list of questions; if you go to the blog category you see a list of articles; both of those are posts that show at the top level, on the list for the category.  Answers are not top-level; they must be attached to questions.  (Articles don't accept answers because they aren't questions.)

Reply:
Okay, my misunderstanding. I'm not sure this vocabulary is completely intuitive but we'll go with it for now and can always change later!

Reply:
It's not ideal and if we can find a better term we should change it, but let's go with this for now.  Assume all our help will be living documents for a while. :-)